near/src/config.rs

```diff
@@ -86,6 +86,9 @@ pub const INITIAL_GAS_LIMIT: Gas = 10_000_000;
 /// Initial gas price.
 pub const INITIAL_GAS_PRICE: Balance = 100;
```

Comment:
let's remove initial gas price. and for genesis use min_gas_price as the gas_price